plugin-helper-overview/api-plugin-helper-service_discovery.md

```diff
@@ -66,5 +74,63 @@ It manages service discovery functionalities such as updating target services an
 
 * [`out_forward`](../output/forward.md)
 
+## Migration guide from `service_discovery_create_manager` to more simpler helper method
+
+Here is the guide to migrate to newer API which is available since v1.13.0.
+
+Example:
+
+```ruby
+require 'fluent/plugin/output'
+
+module Fluent::Plugin
+  class ExampleOutput < Output
+    Fluent::Plugin.register_output('example', self)
+
+    helpers :service_discovery
+
+    def configure(conf)
+      super
+
+      # 1. Remove the following code which parse 'service_discovery' section by yourself
+      #
+      # config = conf.elements(name: 'service_discovery').map do |s|
+      #  { type: :static, conf: s }
+      # end
+
+      # 2. Remove the following code and use service_discovery_configure
+      #
+      # service_discovery_create_manager(
+      #  :out_example_service_discovery_watcher,
+      #  configurations: config,
+      # )
+      service_discovery_configure(
+        :out_example_service_discovery_watcher,
+        static_default_service_directive: 'server'
+      )
+    end
+
+    def write(chunk)
+      # 3. Remove the following code and use service_discovery_select_service to select service
+      #
+      # @discovery_manager.select_service do |node|
+      #  send_data(node, chunk)
+      # end
+      service_discovery_select_service do |node|
+        send_data(node, chunk)
+      end
+    end
+
+    def send_data(node, chunk)
+      # Send data
+    end
+  end
+end
+```
+
+You can also use helper methods such as `service_discovery_services` or `service_discovery_rebalance` instead of
+`@discovery_manager.services` or `@discovery_manager.services`.
```

Comment:
later one is `@discovery_manager.rebalance`?

Reply:
Yes, it should be.